morpheus-server/tests/conftest.py

```diff
@@ -1,12 +1,12 @@
 import pytest
+from app.app import app
 from httpx import AsyncClient
 from morpheus_data.database.database import get_db
-from morpheus_data.models.schemas import User
+from morpheus_data.models.schemas import User, Role
 from morpheus_data.repository.firebase_repository import FirebaseRepository
+from morpheus_data.repository.role_repository import RoleRepository
 from morpheus_data.repository.user_repository import UserRepository
 
-from app.app import app
-
```

Comment:
@juanArias8 this doesn't fit pep rules for imports:

app is a local library and morpheus_data would be a third-party library. This reordering usually happens because the IDE doesn't recognize morpheus_data as 3rd library because it is not in PyPI